js/content/common.js

```diff
@@ -212,6 +212,18 @@ let ExtensionLayer = (function() {
         script.parentNode.removeChild(script);
     };
 
+    // NOTE: use cautiously!
+    // Load remote script in the context of the current tab
+    self.loadInPageContext = function(fun) {
+        return new Promise(function(res) {
+            let script  = document.createElement("script");
+            script.src = fun;
```

Comment:
`fun` is a url? Why is it named "fun"? (Instead of say `src` or `url` or `href` or any of the other variable names that would make it more obvious what it's intended for?

EDIT: we should also filter this to steam's domains. I don't know if it will make it past Google/Firefox review as it is but that might help.